sql/hashtag_statistics.sql

```diff
@@ -1,159 +1,106 @@
-    SELECT ht.hashtag AS tag,
-        ht.id AS hashtag_id,
-        (('hashtag/'::text || ht.hashtag) || '/{z}/{x}/{y}.mvt'::text) AS extent_uri,
-        sum(hashtag_join.buildings_added) AS buildings_added,
-        sum(hashtag_join.buildings_modified + hashtag_join.buildings_deleted) AS buildings_modified,
-        sum(hashtag_join.roads_added) AS roads_added,
-        sum(hashtag_join.road_km_added) AS road_km_added,
-        sum(hashtag_join.roads_modified + hashtag_join.roads_deleted) AS roads_modified,
-        sum(hashtag_join.road_km_modified + hashtag_join.road_km_deleted) AS road_km_modified,
-        sum(hashtag_join.waterways_added) AS waterways_added,
-        sum(hashtag_join.waterway_km_added) AS waterway_km_added,
-        sum(hashtag_join.waterways_modified + hashtag_join.waterways_deleted) AS waterways_modified,
-        sum(hashtag_join.waterway_km_modified + hashtag_join.waterway_km_deleted) AS waterway_km_modified,
-        sum(hashtag_join.coastlines_added) AS coastlines_added,
-        sum(hashtag_join.coastline_km_added) AS coastline_km_added,
-        sum(hashtag_join.coastlines_modified + hashtag_join.coastlines_deleted) AS coastlines_modified,
-        sum(hashtag_join.coastline_km_modified + hashtag_join.coastline_km_deleted) AS coastline_km_modified,
-        sum(hashtag_join.pois_added) AS pois_added,
-        sum(hashtag_join.pois_modified + hashtag_join.pois_deleted) AS pois_modified
-      FROM (hashtags ht
-        JOIN hashtag_join ON ((ht.id = hashtag_join.hashtag_id)))
-      GROUP BY ht.id, ht.hashtag
-    )
-  SELECT without_json.tag,
-     without_json.hashtag_id,
-     without_json.extent_uri,
-     without_json.buildings_added,
-     without_json.buildings_modified,
-     without_json.roads_added,
-     without_json.road_km_added,
-     without_json.roads_modified,
-     without_json.road_km_modified,
-     without_json.waterways_added,
-     without_json.waterway_km_added,
-     without_json.waterways_modified,
-     without_json.waterway_km_modified,
-     without_json.coastlines_added,
-     without_json.coastline_km_added,
-     without_json.coastlines_modified,
-     without_json.coastline_km_modified,
-     without_json.pois_added,
-     without_json.pois_modified,
-     usr_json_agg.users
-    FROM (without_json
-      JOIN usr_json_agg ON ((without_json.hashtag_id = usr_json_agg.hashtag_id)));
+  WITH general AS (
+    SELECT
+      hashtag_id,
+      max(coalesce(closed_at, created_at)) last_edit,
+      count(*) changeset_count,
+      sum(total_edits) edit_count,
+      max(updated_at) updated_at
+    FROM changesets
+    JOIN changesets_hashtags ON changesets.id = changesets_hashtags.changeset_id
+    GROUP BY hashtag_id
+  ),
+  processed_changesets AS (
+    SELECT
+      id,
+      user_id,
+      hashtag_id,
+      measurements,
+      counts
+    FROM changesets
+    JOIN changesets_hashtags ON changesets.id = changesets_hashtags.changeset_id
+  ),
+  user_counts AS (
```

Comment:
I'm not seeing where users 0 and 1 are being ignored here.  Did I miss it?

Reply:
https://github.com/azavea/osmesa-stat-server/pull/20/files#diff-ff794f69c23f9f516fe1dd05a5d23a70R6-R7